<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StitchAdjust</class>
 <widget class="QDialog" name="StitchAdjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
}
QPushButton{
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:18px;
color:rgb(255, 255, 0);
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}
QPushButton::checked{
color:red;
}
QToolButton{
border:noborder;
background-color: transparent;
font-size:24px;
color:blue;
}
QToolButton::checked{
color:red;
}
QLabel{
color:white;
font-size:24px;
background-color:transparent;
}</string>
  </property>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>-
-</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>195</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>+
+</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chainstitch">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>385</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>链
条
压
针</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>385</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setall">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>全
设</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>290</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>参
数
保
存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
